paket_stellar/__init__.py

```diff
@@ -197,6 +198,8 @@ def prepare_escrow(
 
 def new_account(pubkey):
     """Create a new account and fund it with lumens. Debug only."""
+    if not DEBUG:
+        raise AssertionError('creating new account and funding it allowed only in debug mode')
```

Comment:
I don't think we should use AssertionError here (and below). NotImplementedError sounds more suitable.

Reply:
Just saw this in NotImplementedError docs:
> It should not be used to indicate that an operator or method is not meant to be supported at all – in that case either leave the operator / method undefined or, if a subclass, set it to None.

Not sure if we should go for a custom exception or just Exception.

Reply:
Went for custom exception.